src/js/background/assignManager.js

```diff
@@ -113,6 +113,16 @@ const assignManager = {
     return true;
   },
 
+  async handleProxifiedRequest(requestInfo) {
+    if(requestInfo.tabId === -1)
+      return {type: "direct"};
```

Comment:
Move this constant into Utils and use it everywhere, even in comparisons I reckon.
```suggestion
      return Utils.DEFAULT_PROXY;
```

Reply:
Ok I've fixed it, currently it is implemented using `Object.defineProperty` (as otherwise the constant isn't a real constant and can be deleted and modified, e.t.c.)